can/ctypesutil.py

```diff
@@ -42,7 +42,7 @@ def map_symbol(self, func_name, restype=None, argtypes=(), errcheck=None):
                 "Could not map function '{}' from library {}".format(
                     func_name, self._name
                 )
-            )
+            ) from None
```

Comment:
This should be:

```python3
        except AttributeError as attribute_error:
            raise ImportError(
                "Could not map function '{}' from library {}".format(
                    func_name, self._name
                )
            ) from attribute_error
```
Otherwise Traceback is lost, right?

Reply:
Well, this was intentional, since the message is already quite verbose. If you want to commit your proposal, go ahead. ;)

Reply:
If it's intentional then i trust you there :)